WordPress/gutenberg
# A margin option is added to the Embeded preview block

## What?
This PR will add the option to set margins to the embeds.

## Why?
This PR will allow the user to add the margins to the embeds preview.

## How?
The option is added to the inspector control to add the margins to the block.

## Testing Instructions
Here are some steps to check the implementation ;
1.  Add the embedded block to posts or pages.
2.  Add the link to the embed.
3. After adding the link you will notice the additional option to set the margin to the preview.

## Screenshots or screencast 
https://www.loom.com/share/6dbca6117ec6416aad995319736db226
